apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -240,7 +242,13 @@ const DisplayServiceAccountCard = (props: Props) => {
       </FormControl>
       <FormControl marginBottom="none">
         <FormControl.Label marginBottom="none">Status</FormControl.Label>
-        <Paragraph>{isLoading ? <Spinner variant="primary" /> : <RenderStatusInfo />}</Paragraph>
+        <Paragraph>
+          {isLoadingAdminApi && isLoadingDataApi ? (
+            <Spinner variant="primary" />
```

Comment:
i'd recommend using a [Skeleton](https://f36.contentful.com/components/skeleton) component vs a Spinner here, but otherwise LGTM

Reply:
actually i guess this only makes sense when there is going to be an error. in the case where it's only `active` perhaps this doesn't make sense 🤔 ignore this for now.